assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -47,6 +47,7 @@ import {
   NoResults,
   StoryGridView,
 } from '../shared';
+import StoryListView from '../shared/storyListView';
```

Comment:
nit: can you import this with the rest of the shared components right above to avoid the separate import?

Reply:
Ah yes!